Tests/DatabaseAccess.robot

```diff
@@ -0,0 +1,2 @@
+*** Comments ***
+Robot file fo Database Access test scripts
```

Comment:
I believe it is supposed to be 'for' instead of 'fo'

Reply:
This file would be removed